routes/users.py

```diff
@@ -90,7 +90,8 @@ def dict_all(models):
 
         await mailer.send_mail(MailTemplates.VerifyEmail, email, {
             "USER_NAME": user.username,
-            "TOKEN": token
+            "TOKEN": token,
+            "BASE_URL": "http://localhost"
```

Comment:
don't hardcode this, we'll set it via Env vars

Reply:
Yeah, was just there as a test